keno/fuel_pin.inp

```diff
@@ -0,0 +1,47 @@
+=csas6 
+simple fuel pin
+ce_v7_endf
+Read Param 
+npg=100000 gen=110 nsk=10 rnd=19073486328125
```

Comment:
It misses `flx=yes` parameter when compared to the full core model. What does it do?

Reply:
it is just flag for printing the total flux and I removed since we don't need the flux values